frontend/src/component/onboarding/ConnectSDKDialog.tsx

```diff
@@ -106,22 +114,252 @@ const NextStepSectionSpacedContainer = styled('div')(({ theme }) => ({
     padding: theme.spacing(3, 8, 3, 8),
 }));
 
+const SecretExplanation = styled('div')(({ theme }) => ({
+    backgroundColor: theme.palette.background.elevation1,
+    borderRadius: theme.shape.borderRadius,
+    padding: theme.spacing(3),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+}));
+
+const SecretExplanationDescription = styled('div')(({ theme }) => ({
+    backgroundColor: theme.palette.primary.contrastText,
+    borderRadius: theme.shape.borderRadius,
+    padding: theme.spacing(2),
+    flex: 1,
+    color: theme.palette.text.secondary,
+    fontSize: theme.fontSizes.smallBody,
+}));
+
+function splitToken(
```

Comment:
Can this be const splitToken, and can we have test for it?

Reply:
yup, will do